diff a/include/distributions/vendor/fmath.hpp b/include/distributions/vendor/fmath.hpp
--- a/include/distributions/vendor/fmath.hpp
+++ b/include/distributions/vendor/fmath.hpp
@@ -73,7 +73,9 @@
 	#endif
 	#if __GNUC_PREREQ(4, 4) || !defined(__GNUC__)
 		/* GCC >= 4.4 and non-GCC compilers */
-		#include <x86intrin.h>
+    #ifdef __x86_64__
+      #include <x86intrin.h>
+    #endif
 	#elif __GNUC_PREREQ(4, 1)
 		/* GCC 4.1, 4.2, and 4.3 do not have x86intrin.h, directly include SSE2 header */
 		#include <emmintrin.h>
