--- a/examples/mixture/main.py
+++ b/examples/mixture/main.py
@@ -30,6 +30,7 @@
 import numpy
 import scipy
 import scipy.misc
+import imageio
 import scipy.ndimage
 from distributions.dbg.random import sample_discrete, sample_discrete_log
 from distributions.lp.models import nich
@@ -45,7 +46,7 @@
 DATA = os.path.join(ROOT, 'data')
 RESULTS = os.path.join(ROOT, 'results')
 SAMPLES = os.path.join(DATA, 'samples.json.gz')
-IMAGE = scipy.misc.imread(os.path.join(ROOT, 'fox.png'))
+IMAGE = imageio.imread(os.path.join(ROOT, 'fox.png'))
 SAMPLE_COUNT = 10000
 PASSES = 10
 EMPTY_GROUP_COUNT = 10
